### ISSUE TITLE
FEATURE: expose xvfb "nolisten" flag for X screen forwarding

### ISSUE THREAD

**Issue by akhepcat** [ich777/docker-steamcmd-server#70]
for Docker servers like ich777/steamcmd:vrising   that run a virtual frame buffer, it would sometimes be nice to expose that virtual X server upward for debugging purposes.

having an env variable set to disable the '-nolisten tcp'  flag on Xvfb (via /bin/xvfb-run)  by adding "-l"  to the "xvfb-run"  line if the env variable is set.

T'would be a fine addition

**Comment by ich777** [comment]
Can you please tell me what do you want to debug exactly? Just askin because this seems like a very specific issue...
You will still get the log output from V-Rising from `docker logs`

**Comment by akhepcat** [comment]
Ah, i think i was looking at it backward - what i really want is to x11vnc that local Xvfb and expose *that* connectivity.    having the Xvfb listen for tcp connections is the wrong way.

having the x11vnc  running and exposed (when appropriate)  allows the server admin direct visual confirmation of, and access to,  the running game console.

**Comment by ich777** [comment]
Are you really sure that it is possible to issue commands to the server directly in the console, please keep in mind not all dedicated game servers support this.

Wouldn‘t be RCON be a better solution for this use case?

**Comment by akhepcat** [comment]
> Wouldn‘t be RCON be a better solution for this use case?

V Rising only has two implemented RCON commands:  announce  and announcerestart

So having a separate way into the game console in critical for operators that aren't actually running the *game itself*

**Comment by ich777** [comment]
@akhepcat I mean this is no excuse but nobody yet complained about that for 3 years since this container exists.

Let me ask my question again, are you really sure that it is possible to issue commands to the server directly in the console?

...I just tried it and you can't interact with the console, so to speak even if I would tell Xvfb to listen you actually won't be able to interact with the dedicated server since it seems V-Rising is not designed to listen to commands from the dedicated server console window.

**Comment by akhepcat** [comment]
yeah, i don't have any other way to test it, so I don't know.  I was just looking for ways to get access to the admin console without having the game loaded, but if what you're saying is true, there really isn't.

There's so many poor choices in how this dedicated server has been built that it's just frustrating for somebody who grew up managing the og quake servers

**Comment by ich777** [comment]
You just see a console window with all the log output but you actually can't interact with it, sorry...

I'll close this issue, if you need anything else please feel free to post here or create a new issue.
